pwndbg/color.py

```diff
@@ -53,7 +53,7 @@ def get(address, text = None):
     if text is None and isinstance(address, (long, int)) and address > 255:
         text = hex(int(address))
     if text is None:
-        text = int(address)
+        text = str(int(address))
```

Comment:
Agree on `str`, but I'd prefer to print decimal for small numbers.

Perhaps we can add a configuration point for this.  See `gdb.parameter` for more info.